now.json

```diff
@@ -1,7 +1,7 @@
 {
   "rewrites": [
     {
-      "source": "/",
+      "source": "/(api/v1/frameworks)?",
```

Comment:
Is the capture group necessary? I think this would work:

```suggestion
      "source": "/api/v1/frameworks",
```

Reply:
@TooTallNate but we want the group to be optional